server/views/index.pug

```diff
@@ -21,7 +21,7 @@ html
             enableServiceLevelTrends: !{JSON.stringify(enableServiceLevelTrends || false)},
             services: !{JSON.stringify(services || null)},
             enableSSO: !{JSON.stringify(enableSSO || false)},
-            alertsRefreshInterval: !{JSON.stringify(alertsRefreshInterval)}
+            alertsRefreshInterval: !{JSON.stringify(alertsRefreshInterval || -1)}
```

Comment:
THANK YOU!!!!! I was stumped as to why this was failing earlier.

Reply:
@katholen 👍 Sorry for confusion earlier. Configs could use some cleanups, will work on this.